resources/public/api/conf/2.0/application.yaml

```diff
@@ -0,0 +1,10973 @@
+        }
+        </pre>
+                                                                                         </td>
+                                                                                         </tr>
+                                                                                         <tr>
+                                                                                             <td>
+                                                                                                <p>Fund release already exists</p>
+                                                                                                <p class="code--block">
+                                                                                                <strong>lisaManagerReferenceNumber:</strong><br>
+                                                                                                <a href="#section/Testing">Use your test user profile</a><br>
+                                                                                                <br>
+                                                                                                <strong>accountId:</strong><br>0000000409
+                                                                                                </p>
+                                                                                                </td>
+                                                                                                <td>
+                                                                                                    <pre class="code--block">
+        {
+          "eventDate": "2017-05-10",
+          "withdrawalAmount": 4000.00,
+          "conveyancerReference": "CR12345-6789",
+          "propertyDetails": {
+            "nameOrNumber": "1",
+            "postalCode": "AA11 1AA"
+          }
+        }
+        </pre>
+                                                                                                 </td>
+                                                                                                 <td>
+                                                                                                     <p>HTTP status: <code class="code--slim">409 (Conflict)</code></p>
+                                                                                                     <pre class="code--block">
+        {
+          "code": "LIFE_EVENT_ALREADY_EXISTS",
+          "message": "The investor’s life event has already been reported",
+          "lifeEventId": "3456789000"
+        }
+        </pre>
+                                                                                                </td>
+                                                                                                </tr> 
+            </thead>
+            </tbody>
+        </table>
+      operationId: requestReleaseOfFundsToBuyProperty
+      parameters:
```

Comment:
I had a feeling this was wrong as it is missing a Content-Type header which is the same as the RAML as shown in the snippet below.

<img width="637" alt="Screenshot 2023-02-09 at 11 14 10" src="https://user-images.githubusercontent.com/63714619/217797885-bdfc9886-4bc5-4f10-a363-2bb50fceeddd.png">

However, I did some testing to check this and it does require a Content-Type header or it errors as shown in the snippets below.

<img width="1120" alt="Screenshot 2023-02-09 at 11 10 16" src="https://user-images.githubusercontent.com/63714619/217798621-c54f6929-225c-4f4d-86b6-6ce8df26b335.png">

<img width="1125" alt="Screenshot 2023-02-09 at 11 10 37" src="https://user-images.githubusercontent.com/63714619/217798712-f2b63eac-de82-409a-bece-b9cbe694dd55.png">

I think we should correct this at least in this PR by adding the reference `- $ref: '#/components/parameters/contentTypeHeader'`. Thanks

Reply:
1. As far as I know bad request above can be returned only when there was posted empty json body. Please attach full request details including request body and headers. Content type is derived from accept header, so there is no actually need for content type header. It can be sent but it is not important from what I can see. Maybe it was important in initial version of the API.
2. I will double check but this is out of the scope of the ticket. Scope is to migrate RAML to OAS only. 
3. Wider review of headers can be done as separate ticket.

Reply:
This is wrong. This is a Post call. So it requires `Content-Type` header. Why? This is because it has a request body it is used to specify the request body format as stated [here](https://github.com/hmrc/api-documentation-frontend/blob/main/public/common/modules/headers.raml#L17) while accept header is for the response and version [here](https://github.com/hmrc/api-documentation-frontend/blob/main/public/common/modules/headers.raml#L9). I think it is best to jump on a call to go through this. There are some things that are wrong which I ignored but this is one of the ones which I feel should be fixed as it is very misleading.